ontology/bmont.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: ontology_detail
+layout: ontology_detail
+id: bmont
+title: Biomarker Ontology
+browsers:
+- title: BioPortal Ontology Browser
+  label: BioPortal
+  url: [https://bioportal.bioontology.org/ontologies/BMONT]
+contact:
+  email: alpha.tom.kodamullil@scai.fraunhofer.de
```

Comment:
@Astghik-S: Unfortunately this is a blocker. The contact MUST be a person who can be responsive to community requests and is active in the development and maintenance of the ontology. Perhaps you are the right person since you are the ontology submitter and the one authoring the metadata record?